### PR TITLE
Payment service

### PR DESCRIPTION
**Pull request by nadiyafomenko**
- added method for payment for receiving payment status
- removed method for payment refund vecause of inappropriate work

### CODE REVIEW

**Review by Iryna-Bzdyr — APPROVED** [review]

File: typeDefs.js [hunk 1]
```diff
@@ -365,6 +367,7 @@ const typeDefs = gql`
     createNovaPoshtaOrder(data: NovaPoshtaOrderInput): NovaPoshtaOrderResult
     getUkrPoshtaRegion(region: String): UkrPoshtaRegion
     getPaymentCheckout(data: PaymentInput): Payment
+    getPaymentStatus(orderId: String): PaymentStatus
```
Comment (moran711) [inline typeDefs.js:370]: It's make sense to set this field as required.

File: modules/payment/payment.service.js [hunk 2]
```diff
@@ -34,48 +36,51 @@ class PaymentService {
...
     });
+
     const requestData = {
       order_id: orderId,
       order_desc: orderDesc,
       currency,
       amount,
     };
 
+    console.log(requestData);
```
Comment (dimaPalii) [inline modules/payment/payment.service.js:47]: Please, clean-up code before merge

File: modules/payment/payment.service.js [hunk 3]
```diff
@@ -21,8 +22,9 @@ class PaymentService {
         }
       });
     const signString = secret + '|' + Object.values(ordered).join('|');
+
     return crypto
-      .createHash(CRYPTO)
+      .createHash('sha1')
```
Comment (dimaPalii) [inline modules/payment/payment.service.js:27]: Let's use more secure SHA-256 algorithm

File: modules/payment/payment.graphql.js [hunk 4]
```diff
@@ -6,6 +6,16 @@ const paymentType = `
   }
 `;
 
+const paymentStatus = `
+  type PaymentStatus {
+    orderId: String!
+    orderStatus: String!
+    orderTime: String!
+    amount: String!
+    currency: String
```
[the remark this answers is not in the document]
Comment (nadiyafomenko) [reply modules/payment/payment.graphql.js:15]: No, payment system require one particular currency

File: typeDefs.js (hunk 1 above)
Comment (nadiyafomenko) [reply typeDefs.js]: fixed

File: modules/payment/payment.service.js (hunk 2 above)
Comment (nadiyafomenko) [reply modules/payment/payment.service.js]: fixed

**Review by moran711 — APPROVED** [review]

File: modules/payment/payment.service.js (hunk 3 above)
Comment (nadiyafomenko) [reply modules/payment/payment.service.js:27]: for this reason, Fondy requieres sha1 signature coding

**Review by dimaPalii — APPROVED** [review]

**Pull request merged**